### ISSUE TITLE
Restarting the service from the program 

### ISSUE THREAD

**Issue by pr200sd** [maximkulkin/esp-homekit-demo#422]
Good day.

Is it possible to orbinize the restart of the service, without restarting esp32, for example, after erasing the data, I want to be able to pair with the phone again, now I am doing a power reset for this.

**Comment by maximkulkin** [comment]
No, it is not possible at the moment. Implementation is not hard, but I couldn't find any practical use case for this.

**Comment by pr200sd** [comment]
Got it, thanks for the answer. My use case is as follows: after removing the accessory on the phone and erasing the data in esp32, a new device cannot be added without restarting esp32.

**Comment by rottenhowler** [comment]
So, what's the problem restart the device?

**Comment by maximkulkin** [comment]
Then restart the device.

**Comment by pr200sd** [comment]
There are no problems to reboot the device, but my esp is used as a gateway for a multichannel relay and controls channels via modbus, https://youtu.be/7YfrBtO3Dos?t=144
and in some cases information will be lost and the relay will be reset. In principle, I have already taken into account this point and now all states are remembered before rebooting, so there are no problems.
